x-pack/filebeat/input/s3/config.go

```diff
@@ -42,16 +44,22 @@ func (c *config) Validate() error {
 		return fmt.Errorf("visibility timeout %v is not within the "+
 			"required range 0s to 12h", c.VisibilityTimeout)
 	}
+
```

Comment:
is this intentional?

Reply:
yes :) I just want to make it look prettier 😂

Reply:
👍